.github/workflows/scala-steward.yml

```diff
@@ -1,19 +1,25 @@
 name: Scala Steward
 
-# This workflow will launch everyday at 00:00
+# This workflow will launch every day at 00:00
 on:
   schedule:
     - cron: '0 0 * * *'
   workflow_dispatch: {}
 
+permissions:
+  contents: write
+  pull-requests: write
+
 jobs:
   scala-steward:
     timeout-minutes: 45
     runs-on: ubuntu-latest
     name: Scala Steward
     steps:
+      - name: Setup sbt
+        uses: sbt/setup-sbt@v1
```

Comment:
This is new to me 😮

Reply:
It's quite new. It's has been implemented as the more recent Ubuntu-based Github runners don't have sbt installed by default anymore